src/sig_stateful/lms/Makefile

```diff
@@ -0,0 +1,879 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
Is it intentional/necessary to add `cmake` generated files into git?

Reply:
Nope. A miss. Will remove.